docs/sources/reference/components/prometheus/prometheus.exporter.snmp.md

```diff
@@ -17,25 +17,34 @@ The `prometheus.exporter.snmp` component embeds
 
 ## Usage
 
+### Recommended usage
+
 ```alloy
 prometheus.exporter.snmp "LABEL" {
   config_file = SNMP_CONFIG_FILE_PATH
-
-  target "TARGET_NAME" {
-    address = TARGET_ADDRESS
-  }
+  targets     = TARGET_LIST
 }
 ```
 
-or
+### Deprecated usage
```

Comment:
We could use this example: https://grafana.com/docs/writers-toolkit/write/shortcodes/#boltdb-deprecated to document the deprecated target block.

And I'd suggest we move this information down to the actual target block section (line 93) since that's where users will encounter it. And... don't provide a deprecated Usage section... Explicitly showing how to use a deprecated bit encourages people to keep using it.